case/step/case_model.py

```
@@ -0,0 +1,559 @@
+    return max(_wires(fn), key=lambda w: (lambda b: b.size.X * b.size.Y)(w.bounding_box()))
+
+def centered_face(fn):
+    """OpenSCAD import(center=true): outer wire -> face, recentre on its bbox."""
+    f = make_face(_outer_wire(fn))
+    b = f.bounding_box()
+    return f.moved(Location((-(b.min.X + b.max.X) / 2, -(b.min.Y + b.max.Y) / 2, 0)))
+
+def centered_faces_all(fn):
+    """All wires -> faces, recentred by the COMBINED bbox (multi-hole svgs)."""
+    ws = _wires(fn)
+    mnx = mny = 1e9; mxx = mxy = -1e9
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)